.travis.yml

```diff
@@ -63,6 +65,7 @@ script:
 
 before_deploy:
   - export GEOPYSPARK_VERSION_SUFFIX="-${TRAVIS_COMMIT:0:7}"
+  - aws s3 rm s3://geopyspark-dependency-jars/geotrellis-backend-assembly-*.jar
```

Comment:
We shouldn't delete all the versions of the file. Likely in near future we will be in position that a couple of versions are being used, stable and head

Reply:
:+1: